compiler/rustc_hir_analysis/src/hir_ty_lowering/mod.rs

```diff
@@ -2025,23 +2040,138 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
     ///
     /// Early-bound const parameters get lowered to [`ty::ConstKind::Param`]
     /// and late-bound ones to [`ty::ConstKind::Bound`].
-    pub(crate) fn lower_const_param(&self, hir_id: HirId) -> Const<'tcx> {
```

Comment:
how come this was changed. I think it would be nicer to keep it the same as it was since `lower_ty_param` also doesn't take a `DefId`